src/components/controls/Controls.js

```diff
@@ -2,22 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './Controls.css';
 
-const Controls = ({ actions, handleSelection }) => (
+const Controls = ({ dispatch }) => (
```

Comment:
after reading your prop types, I understand that this is an array, but naming it dispatch is probably not the best convention